examples/index.html

```diff
@@ -80,6 +80,11 @@
             <img src='screenshots/gpx.jpg'>
             <iframe src="" style="display: none;"></iframe>
         </div>
+        <div class="exampleContainer">
+            <h3><a href='./positionGlobe.html' target='_blank'>Position a mesh</a></h3>
+            <img src='screenshots/globe.jpg'>
```

Comment:
Can you provide a screenshots of the initial view of your example?

Reply:
yes, it's done.